app/src/main/java/com/cornellappdev/volume/ui/viewmodels/PublicationsViewModel.kt

```diff
@@ -20,11 +26,15 @@ class PublicationsViewModel @Inject constructor(
     private val userPreferencesRepository: UserPreferencesRepository,
     private val userRepository: UserRepository,
     private val publicationRepository: PublicationRepository
+
 ) :
     ViewModel() {
 
+    private val _setOfPubsFollowed = mutableSetOf<String>()
+
     data class PublicationsUiState(
-        val publicationsState: PublicationsRetrievalState = PublicationsRetrievalState.Loading
+        val followedPublicationsState: PublicationsRetrievalState=PublicationsRetrievalState.Loading,
+        val unfollowedPublicationsState: PublicationsRetrievalState=PublicationsRetrievalState.Loading
```

Comment:
Change name from unfollowedPublicationsState to morePublicationsState. The name change is more descriptive of what the publications are being used for!